webpack.config.js

```diff
@@ -33,11 +34,62 @@ module.exports = {
     devtool: "source-map",
     externals: [ "mendix/lang", "react", "react-dom" ],
     plugins: [
+        new CleanWebpackPlugin("dist/tmp"),
         new CopyWebpackPlugin([
-            { from: "src/**/*.xml" },
+            { from: "src/**/*.xml", copyUnmodified:true },
             { from: "src/**/*.png" }
         ], { copyUnmodified: true }),
         new ExtractTextPlugin({ filename: "./src/com/mendix/widget/custom/GoogleMaps/ui/[name].css" }),
         new webpack.LoaderOptionsPlugin({ debug: true })
     ]
 };
+
+const googleMapsConfig = {
```

Comment:
name `previewConfig` `previewContextConfig` would be better of the top one is named `widgetConfig`